### PR TITLE
[cxxmodules] Load core C++ modules

### PR DESCRIPTION
**Pull request by Teemperor**

### CODE REVIEW

File: core/meta/inc/TInterpreter.h [hunk 1]
```diff
@@ -162,6 +162,7 @@ class TInterpreter : public TNamed {
                                void (*histaddFunc)(const char* line)) = 0;
    virtual void     Reset() = 0;
    virtual void     ResetAll() = 0;
+   virtual void     SetupCoreModules() = 0;
```
Comment (Axel-Naumann) [inline core/meta/inc/TInterpreter.h]: `TCling` doesn't seem to implement this? Is this interface actually needed, or can this (assuming "this" is really `LoadCoreModules()`) be a private function of `TCling` or even a static function in TCling.cxx?

Comment (Teemperor) [reply core/meta/inc/TInterpreter.h]: Valid point, that was an artifact of the first version of this patch. Thanks! It's now a static function in TCling.

**Comment by Teemperor** [comment]
@phsft-bot build with flags -Druntime_cxxmodules=On

**Comment by Teemperor** [comment]
@phsft-bot build with flags -Druntime_cxxmodules=Off

Just to make Jenkins green

**Comment by vgvassilev** [comment]
@phsft-bot build just on windows10/vc15

**Pull request merged**